packages/react/src/components/dynamic-flow/configured-handle-form.tsx

```diff
@@ -1,24 +1,24 @@
 import { Factor } from "@slashid/slashid";
```

Comment:
Moved this file to `dynamic-flow` - the scope of this ticket blew up, so I had to make a cut here. A follow up task would be to reimplement the DynamicFlow internally using the slots/composition API.